app/src/main/java/com/example/lapnguyen/weather_3/SettingsActivity.java

```diff
@@ -0,0 +1,18 @@
+package com.example.lapnguyen.weather_3;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.preference.PreferenceActivity;
+import android.support.v7.app.AppCompatActivity;
```

Comment:
ERROR: Unused import: android.support.v7.app.AppCompatActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports